tests/e2e/do_nothing_test.go

```diff
@@ -1,9 +0,0 @@
-package e2e_test
```

Comment:
if you drop this file then the setup-full-cluster target in the makefile fails as it can find any tests with that filter. Please restore it back.

Reply:
wait, it doesnt? Im pretty sure it failed for me when I was creating this....anyway, then its good!

Reply:
ah no, the problem here is that it doesnt really setup the system as it doesnt install the deps, rancher and the operator, which is the point of the target, to get a full deployed system with no tests run, so its pristine.

Reply:
it does install it, the `BeforeSuite` block in `e2e_suite_test.go` installs all dependencies